tests/zeek/file-analysis-data-in.zeek

```diff
@@ -42,4 +48,15 @@ protocol analyzer spicy::SSH over TCP:
     port 22/tcp,
     replaces SSH;
 
+on SSH::Banner::software -> event have_filename($file, self.file_name);
```

Comment:
We will need a doc update in `spicy/spicy/doc/zeek.rst` for that: remove the caveat that `$file` is only available in file analyzers.